### DISCUSSION TITLE
How to use Web3j  in Java back-end?

### DISCUSSION THREAD

**Discussion by Pmrcvjw** [web3j/web3j#1762] [Q&A]
I'm a beginner with Web3J library. Now I met a new test scene that some data is required to be sent to the Java back-end to upload to the alliance chain (e.g., HyperLedger BeSu). This means the user signed data by metaMask in Vue front-end first should be sent to the Java back-end and then re-assembled as a required message and uploaded to the alliance chain through Web3J sdk method! However, for this change, i still haven't find a solution to build such a maven project with Web3J sdk, who can give some suggestions or links about this question such that i could follow. I'm grated for your helps.

**Comment by wlswo** [comment]
https://github.com/web3j/web3j-maven-plugin
